.github/workflows/nightly-test.yml

```diff
@@ -0,0 +1,53 @@
+name: nightly test
+
+on:
+  schedule:
+    - cron: '00 18 * * *'  # At UTC 18:00 AM, everyday, use https://crontab.guru/
```

Comment:
@szha Do you have any suggestions on which time should the nightly test be triggered. I'm not sure what exact time will the MXNet release be accessible from dist.mxnet.io.